content/w/santino.md

```diff
@@ -1,12 +1,56 @@
 +++
 title = "Santino"
 template = "talent_page.html"
+authors = ["Sewi The Referee"]
 [taxonomies]
 country = ["IT"]
 [extra]
 career_aliases = ["Santino Marella"]
 +++
 
+Santino Marella is a canadian professional wrestler best known from his time at WWE. In polish wrestling scene he is also known from his apperances at [Prime Time Wrestling](@/o/ptw.md)
+
+## Basic info
+
+* Years active: 2002-present
+* Gimmicks/nicknames:
+  - Santino Marella {{ org_badge(orgs=['wwe']) }}
+  - Santino {{ org_badge(orgs=['ptw']) }}
+* Trained by:
+  - Don Kolov
+  - OVW Wrestling School
+ 
+## Professional wrestling career
+
+### World Wrestling Entertainment (2012)
+
+In April 2012 Santino Marella made his apperance at [WWE RAW House Show](@/e/wwe/2012-04-12-wwe-raw-house-show.md) which took place in ERGO Arena in Tri-City. It was one of few times when WWE hosted a live event in Poland.
+
+### Prime Time Wrestling (2021-2024)
+
+Santino made his return to polish soil after nearly a decade when he appeared at [Prime Time Wrestling's](@/o/ptw.md) first show [REVOLUCJA](@/e/ptw/2021-10-09-ptw-1-revolucja.md). He alligned himself with [Disco Pablo](@/w/disco-pablo.md) to be defeated by [Syriusz Dziedzic](@/w/dziedzic.md) and Marius Al-Ani. 
+
+He later made few other apperances, mostly to provide english live commentary along with Lewis Costello. Besides that, at [PTW #3: Legends](@/e/ptw/2022-11-26-ptw-3-legends.md) it was announced that Santino will be a co-owner of PTW. Moments later he was attacked by [Bad Bones](@/w/bad-bones.md). It is confirmed that whole angle with co-ownership of PTW by canadian wrestler was a work, and due to lack of regular apperances of Santino afterwards the angle was eventually scrapped.
```

Comment:
Te wszystkie "will", które zamieniałem na "would", to przykład tzw. tense shift występującego w trybie niebezpośrednim (reported speech). W skrócie, cofasz czasowniki o jeden czas: "He says he will go" -> "He said he would go".